dali/python/nvidia/dali/plugin/pytorch.py

```diff
@@ -78,13 +78,19 @@ class DALIGenericIterator(object):
                  iterator == 'size'. Setting this flag to False will
                  cause the iterator to return the first integer multiple
                  of self._num_gpus * self.batch_size which exceeds 'size'.
+    dynamic_shape: bool, optional, default = False
+                 Whether the shape of the output of the DALI pipeline can
+                 change during execution. If True, the mxnet.ndarray will be resized accordingly
```

Comment:
Probably `pytorch tensor` instead of `mxnet.ndarray` here

Reply:
Good catch